.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 
+branches:
+  only: 
+   - master
+
```

Comment:
This got lumped in here, build only PR's and master branch, avoids duplicate builds when working on a branch in the repository.